pkg/storage/system.go

```diff
@@ -122,3 +126,35 @@ func (s *Store) listSystems(ctx context.Context) ([]*models.System, error) {
 
 	return results, nil
 }
+
+const sqlFetchSystemByIntakeID = `
+	SELECT
+		id,
+		project_name AS name
+	FROM system_intake
+	WHERE
+		status='LCID_ISSUED' AND
+		request_type='NEW' AND
+		lcid IS NOT NULL AND
+		id = $1;
+`
+
+// FetchSystemByIntakeID queries the DB for a single system
+func (s *Store) FetchSystemByIntakeID(ctx context.Context, intakeID uuid.UUID) (*model.System, error) {
```

Comment:
WHOA! I hadn't considered the possibility of having the GQL-generated `model` types be what is returned from our `storage` layer. 🤯 
My initial gut reaction is that it gives me the willies... But I'll need to kick that around a bit to see if that's just my internal Negative Nancy acting up.

Reply:
I'm not sure what the best way to handle this is, either. I was having a hard time explaining to myself what the purpose of the `models` package was, how the `model` package fits in, etc. Partly we might need less generic names for them, but in any case this is something we should sort out as a team because 🤷🏻 .

Reply:
Can we give this a go and see if it saves us writing some code by hand for now? Seems like we can break out of this pattern later if needed, but it's ok to have as an initial pass.